src/routes/transactions/__tests__/get-multisig-transactions.e2e-spec.ts

```diff
@@ -0,0 +1,123 @@
+    const cacheKey = `${chainId}_${safeAddress}_multisig_transactions`;
+    const cacheKeyField = `-modified_undefined_true_${executionDateGte}_${executionDateLte}_undefined_undefined_undefined_undefined_undefined`;
+    const expectedResponse = getJsonResource(
+      'e2e/native-token-expected-response.json',
+    );
+
+    await request(app.getHttpServer())
+      .get(
+        `/chains/${chainId}/safes/${safeAddress}/multisig-transactions?execution_date__gte=${executionDateGte}&execution_date__lte=${executionDateLte}`,
+      )
+      .expect(200)
+      .then(({ body }) => {
+        expect(sortTransactionsByTimestamp(body)).toEqual(
+          sortTransactionsByTimestamp(expectedResponse),
+        );
+      });
+
+    const cacheContent = await redisClient.hGet(cacheKey, cacheKeyField);
+    expect(cacheContent).not.toBeNull();
+  });
+
+  it('GET /safes/<address>/multisig-transactions (ERC-20 + Settings change)', async () => {
+    const safeAddress = '0xCe95F1F4ACADEe697993B0E3a0FE48B444679046';
+    const executionDateGte = '2022-12-01T00:00:00.000Z';
+    const executionDateLte = '2022-12-07T11:00:00.000Z';
+    const cacheKey = `${chainId}_${safeAddress}_multisig_transactions`;
+    const cacheKeyField = `-modified_undefined_true_${executionDateGte}_${executionDateLte}_undefined_undefined_undefined_undefined_undefined`;
+    const expectedResponse = getJsonResource(
+      'e2e/erc20-expected-response.json',
+    );
+
+    await request(app.getHttpServer())
+      .get(
+        `/chains/${chainId}/safes/${safeAddress}/multisig-transactions?execution_date__gte=${executionDateGte}&execution_date__lte=${executionDateLte}`,
+      )
+      .expect(200)
+      .then(({ body }) => {
+        expect(sortTransactionsByTimestamp(body)).toEqual(
+          sortTransactionsByTimestamp(expectedResponse),
+        );
+      });
+
+    const cacheContent = await redisClient.hGet(cacheKey, cacheKeyField);
+    expect(cacheContent).not.toBeNull();
+  });
+
+  it('GET /safes/<address>/multisig-transactions (ERC-721)', async () => {
+    const safeAddress = '0x4127839cdf4F73d9fC9a2C2861d8d1799e9DF40C';
+    const executionDateGte = '2022-11-20T00:00:00.000Z';
+    const executionDateLte = '2022-12-06T00:00:00.000Z';
+    const cacheKey = `${chainId}_${safeAddress}_multisig_transactions`;
+    const cacheKeyField = `-modified_undefined_true_${executionDateGte}_${executionDateLte}_undefined_undefined_undefined_undefined_undefined`;
+    const expectedResponse = getJsonResource(
+      'e2e/erc721-expected-response.json',
+    );
+
+    await request(app.getHttpServer())
+      .get(
+        `/chains/${chainId}/safes/${safeAddress}/multisig-transactions?execution_date__gte=${executionDateGte}&execution_date__lte=${executionDateLte}`,
+      )
+      .expect(200)
+      .then(({ body }) => {
+        expect(sortTransactionsByTimestamp(body)).toEqual(
+          sortTransactionsByTimestamp(expectedResponse),
```

Comment:
Yes, it should, I experienced divergences because of different `modified` timestamps between Goerli staging and production services, but it shouldn't be necessary anymore 👍 I'll remove the sorting.

Reply:
Changed in [85720bd](https://github.com/5afe/safe-client-gateway-nest/pull/197/commits/85720bdf33a94eaab2d886ffe53d1e0f35df0059)